azkaban-common/src/main/java/azkaban/executor/ExecutionReference.java

```diff
@@ -18,16 +18,22 @@
 
 public class ExecutionReference {
```

Comment:
not sure if it is true but it seems like we no longer need this ExecutionReference any more, the new executor IS in the concept of this executionReference so wrapping up again the executor is a little bit over.